documentation/Realtime_Hooks.md

```
@@ -0,0 +1,850 @@
+      mountCount -= 1;
+      if (mountCount <= 0) {
+        disconnect();
+      }
+    };
+  }, []);
+}
+```
+
+**Scenario**: Three tables mounted simultaneously
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)